README.md

```diff
@@ -19,29 +21,27 @@ If you don't want to configure things, you can download a repository with Vue 3,
 
 ## Docs
 
-Docs are located in [this repo](https://github.com/vuejs/vue-test-utils-next-docs). Read them [here](https://vuejs.github.io/vue-test-utils-next-docs/guide/introduction.html). They are in a separate repository because running Vuepress alongside a repo with Vue 3 causes conflicts - Vuepress expects to be running against Vue 2. This seems like the most simple solution for now.
+Docs are located in [this repo](https://github.com/vuejs/vue-test-utils-next-docs). Read them [here](https://vue-test-utils.vuejs.org/v2/). They are in a separate repository because running Vuepress alongside a repo with Vue 3 causes conflicts - Vuepress expects to be running against Vue 2. This seems like the most simple solution for now.
 
 ## Development
 
 It's a pretty small codebase at the moment. Get started by running `yarn install`. You can run the tests with `yarn test`. That's it!
 
-There is a lot of work to do. See issues for some basic TODOs, or the table at the bottom of this page.
+There is still some work left to do. See issues for some basic TODOs, or the table at the bottom of this page.
 
 ## What works?
 
-See the [docs](https://vuejs.github.io/vue-test-utils-next-docs/guide/introduction.html). Most basic DOM interactions work. Advanced features like `shallowMount` and `stubs` are a work in progress.
+See the [docs](https://vuejs.github.io/vue-test-utils-next-docs/guide/introduction.html).
 
 ## Contributing
 
-This is still very much alpha - if you want to add a feature, have a hack or ping someone in Discord to chat, or check out the issues and project board.
-
-## Where is shallowMount
+This is still quite alpha, but we plan on moving to RC sooner than later. If you want to add a feature, have a hack or ping someone in Discord to chat, or check out the issues and project board.
 
-It's coming soon. We are working on it. Stay tuned!
+There's also some [work left to do in docs](https://github.com/vuejs/vue-test-utils-next-docs/issues).
 
 ## Comparsion with Vue Test Utils beta (targeting Vue 2)
 
-This is table for those coming from VTU beta, comparing the two APIs. A lot of things are still a work in progress.
+This is table for those coming from VTU 1, comparing the two APIs. Some things are still a work in progress.
```

Comment:
Should we just remove this table? Perhaps we should add this info to the migration guide.

Reply:
yeah, thought about that, too. Makes sense. Let's do this: keep it here for now, and I'll submit a PR tomorrow or so in docs to port it there :)